app/transport/rest/contracts.go

```diff
@@ -6,5 +6,6 @@ import (
 
 // UserUsecase represents User use-case layer.
 type UserUsecase interface {
-	Signup(entities.User) (string, error)
+	SignUp(entities.User) (string, error)
```

Comment:
Why don't we change `Signup` method as well?

Reply:
Becouse we have no other usecases now. When we will add some entity like admin, we`ll add AdminUsecase interface